blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { Carousel, Image, Icon } from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
+
+// import { useFusionContext, useAppContext } from "fusion:context";
+// import getProperties from "fusion:properties";
+// import getThemeStyle from "fusion:themes";
+// import getTranslatedPhrases from "fusion:intl";
+const BLOCK_CLASS_NAME = "b-story-carousel";
+
+const StoryCarousel = ({
+	arcSite,
+	customFields: {
+		carouselContentConfig: { contentService = "", contentConfigValues = {} } = {},
+		headline,
+		carouselLabel = "Carousel of Images",
+		identifier,
+		rightArrowLabel,
+		leftArrowLabel,
+	},
+}) => {
+	// const { locale = "en" } = getProperties(arcSite);
+	// const phrases = getTranslatedPhrases(locale);
+	const content = useContent({
+		source: contentService,
+		query: contentConfigValues,
+	});
```

Comment:
PageBuilder is giving errors around `content_elements` this is because later in the block you are destructing the object but in some cases there is no object - we should set an OR to a blank object to stop further code erroring

```suggestion
	}) || {};
```